templates/credentials/base_cred.html

```diff
@@ -1,10 +1,14 @@
 {% extends "templates/base.html" %}
 
 {% block meta_copydoc %}
-    https://drive.google.com/drive/folders/1XqlPwl4su8nwIjbnhbfTpG5vJOl8JLxI
+  https://drive.google.com/drive/folders/1XqlPwl4su8nwIjbnhbfTpG5vJOl8JLxI
 {% endblock meta_copydoc %}
 
+{% block body_class %}
```

Comment:
I suppose this is enforced by the formatter that blocks are not in single line, but in cases like this one, when it's only a class name, or above, where it's a copydocs link it makes them multiline in resulting HTML.

<img width="351" alt="image" src="https://github.com/canonical/ubuntu.com/assets/83575/b769b43a-5fa8-41d0-a11f-3dc9ed0d6929">

<img width="718" alt="image" src="https://github.com/canonical/ubuntu.com/assets/83575/41420566-13ed-4334-b9aa-95b784bf9f96">

It's not invalid, and probably will work fine in most cases, but it would be better to make it not have unnecessary line breaks or whitespace inside these values.

Although, I guess it's not necesarily in scope of this PR.
Maybe values like these should be variables in the template, not blocks.

Reply:
After a quick investigation, I believe the true fix is to update the template-finder project to configure Jinja to `trim_blocks`, the first newline after a template tag is removed automatically. Then we can use the whitespace control to trim further whitespace using `{%- block -%}`.

Out of scope for this PR.

Reply:
Issue created: https://github.com/canonical/ubuntu.com/issues/13994